linode/manager
# refactor: [M3-7897] - Query Key Factory for Regions

## Description 📝

Uses our new _query key factory_ for **region queries** 🗺️ 

## How to test 🧪

- Check for any regressions with regions and region availability
  - For example, check the plans table on the Linode Create page 

## As an Author I have considered 🤔
